src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -22,24 +38,42 @@
   private final Joystick m_driverJoystick;
   private final MotorController[] m_motorControllers;
   private final DifferentialDrive m_differentialDrive;
-  //public static ADIS16448_IMU m_gyro; Non-native gyro, might use later
-  public static ADXRS450_Gyro m_gyro;
+  private DifferentialDrivetrainSim m_DifferentialDrivetrainSim;
+  public final Field2d m_field = new Field2d();
+  private final double KvLinear = 1.98;
+  private final double KaLinear = 0.2;
+  private final double KvAngular = 1.5;
+  private final double KaAngular = 0.3;
+  private AnalogGyro m_gyro = new AnalogGyro(1);
+  private AnalogGyroSim m_gyroSim = new AnalogGyroSim(m_gyro);
```

Comment:
Put the instantiation (right side of the = ) to the Robot.isSimulation section in the constructor

Reply:
done